docs/news/daos.md

```diff
@@ -16,3 +16,10 @@ In my last article, I’ve discussed some of the most common types of blockchain
 _Author [Matan Field](https://medium.com/@matanfield) on Feb 6_ | _Author [Cris C.](https://blog.goodaudience.com/@carrascosa.cobos) on Jan 2_ |  |
 **Blockchain Is a Decentralized Governance System.** Blockchain hype is at an all-time high and many people anticipate the first decentralized application to reach the market and gain mass adoption. The term DAO (decentralized autonomous organization) is no longer esoteric. And hundreds of like-minded teams are building decentralized collaborative social news, insurance and investment platforms, numerous collaboration spaces, and even a couple of decentralized autonomous space agencies. | Since the “hack” of The DAO, this organizational structure has not been much used, I believe partially because of the reputational damage it caused, and partially due to its intrinsic complexity, especially for those who are not familiar with decentralized governance and/or blockchain. Nevertheless, if 2017 has been the year for ICOs, 2018 may be the year for the DAO return. |  |
 [Read More](https://medium.com/daostack/decentralized-governance-first-principles-1fc6eaa492ed)| [Read More](https://blog.goodaudience.com/2018-the-dao-returns-5868a473afb0) |  |
+| [**Malta Blockchain Regulation Proposal: Legal Personality for DAOs and Smart Contracts**](https://chainstrategies.com/2018/03/11/malta-blockchain-regulation-proposal-legal-personality-for-daos-and-smart-contracts/) |
+|:-----------:|
+[_DAO_](tag.md) |
+![Alt tcsdsdsext](https://chainstrategies.files.wordpress.com/2018/03/20180311-chain-2130988.jpg)  |
```

Comment:
Weird alt text and doesn't follow the previous format of adding an image, but works.